example.bc

```diff
@@ -78,7 +78,10 @@ option "render_commas" "True"
      'loss_threshold': 50,
      'wash_pattern': 'Assets:Investments',
   },
-
+  'performance': {
+     'value': ['Assets:Investments'],
```

Comment:
Can we call this 'accounts_patterns' for consistency with the other modules?
Also, can we accept both a list and a single string so it doesn't trip people up (other modules accept a string, not a list).